scripts/test_install.sh

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# scripts/test_install.sh — smoke-tests the install.sh resolver in isolation.
+set -euo pipefail
+
+REPO_ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+# Use a fixture latest.json that mirrors what the real release publishes.
+FIXTURE="$REPO_ROOT/scripts/fixtures/latest.json"
+
+# The resolver function should be sourced, not invoked end-to-end (no curl).
+if ! source "$REPO_ROOT/scripts/install.sh" --source-only 2>/dev/null; then
```

Shellcheck (0.11.0)

[info] Not following: ./scripts/install.sh was not specified as input (see shellcheck -x).

(SC1091)